2022-07_sprint/sprint_parties/media_kit.md

```diff
@@ -0,0 +1,56 @@
+# Social Media Kit
+
+(media_kit)=
+
+## Event Hashtag
+
+#DataUmbrellaPyMCSprint
+
+## Social Media Accounts
```

Comment:
The table is not properly aligned and pymc twitter and discourse have incorrect alt text: https://pymc-data-umbrella--102.org.readthedocs.build/en/102/2022-07_sprint/sprint_parties/media_kit.html#social-media-accounts

Reply:
Fixed